extension/src/frontend/view/ToolBox.tsx

```diff
@@ -54,6 +57,11 @@ export function ToolBox({
         border: 1px solid var(--gray-6);
       `}
     >
+      <Tooltip delayDuration={0} asChild content="Stop recording">
+        <Toolbar.Button onClick={onStopRecording}>
+          <StopIcon />
+        </Toolbar.Button>
+      </Tooltip>
```

Comment:
Any thoughts on adding a separator between the Stop and the Inspect buttons?

Reply:
Agree with Cristiano. ☝️

Reply:
Oh, it "got lost" when I moved the button to the left. Thanks a lot!